pkg/k8s/apis/cilium.io/v2/register_test.go

```diff
@@ -16,6 +16,7 @@ package v2
 
 import (
 	. "gopkg.in/check.v1"
+	"regexp"
```

Comment:
We usually change the order of imports, and always group the stdlib imports first.